test/model.translateAliases.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Test dependencies.
+ */
+
+var start = require('./common'),
+    assert = require('power-assert'),
+    mongoose = start.mongoose;
+
+describe('model translate aliases', function() {
+  it('should translate correctly', function() {
+    var Character = mongoose.model('Character', new mongoose.Schema({
+      name: { type: String, alias: '名' },
+      bio: {
+        age: { type: Number, alias: '年齢' }
+      }
+    }));
+
+    assert.deepEqual(
+      // Translate aliases
+      Character.translateAliases({
+        '名': 'Stark',
+        '年齢': 30
+      }),
+      // How translated aliases suppose to look like
+      {
```

Comment:
My bad, I was over thinking it  
Do you prefer that I put it under schema instead of model?